### PR TITLE
Clean up README

### PR DESCRIPTION
**Pull request by Moult**
This new proposed readme makes it more obvious to readers that there is a large collection of tools available, a brief glance of what they do, and directs users to the dedicated websites and documentation for further instructions on installation and so on.

### CODE REVIEW

**Comment by aothms** [comment]
Yeah good point if the README becomes more general purpose (such as pypi) then it should be a bit less specific. Two comments.

File: README.md [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
 
-For more information, see
-* [http://ifcopenshell.org](http://ifcopenshell.org)  
-* [http://academy.ifcopenshell.org](http://academy.ifcopenshell.org)
+IfcOpenShell is an open source ([LGPL]) software library for working with Industry Foundation Classes ([IFC]). Complete
+parsing support is provided for [IFC2x3 TC1], [IFC4 Add2 TC1], IFC4x1, IFC4x3, and IFC4x3. Extensive geometric support
+is implemented for the IFC releases [IFC2x3 TC1] and [IFC4 Add2 TC1]. Extending with support for arbitrary IFC schemas
+is possible at compile-time when using C++ and at run-time when using Python.
+
+IfcOpenShell comes with an ecosystem of tools, notably including IfcConvert (an application to convert IFC models to
```
Comment (aothms) [inline README.md:10]: In addition to a C++ and Python API, IfcOpenShell comes with ... ?

File: README.md [hunk 2]
```diff
@@ -19,307 +28,41 @@ For more information, see
 | Sponsor development on OpenCollective           | [![Financial Contributors](https://opencollective.com/opensourcebim/tiers/badge.svg)](https://opencollective.com/opensourcebim/)             |
 | Docker hub                                      | [![Docker Pulls](https://img.shields.io/docker/pulls/aecgeeks/ifcopenshell)](https://hub.docker.com/r/aecgeeks/ifcopenshell)                 |
 
-Prerequisites
```
Comment (aothms) [inline README.md:22]: For the removed sections can we have direct links to some of the doc content for:

- All building/installing on X
- Quick start

**Review by aothms — CHANGES_REQUESTED** [review]

**Comment by Moult** [comment]
Changes made :)

**Pull request merged**